earth_enterprise/src/common/serverdb/serverdbReader.cpp

```diff
@@ -29,6 +29,55 @@
 #include "common/geGdalUtils.h"
 #include "common/generic_utils.h"
 
+
+std::string JsToJson(const std::string& in)
```

Comment:
put this in an anonymous namespace to emphasize it is only used within this file.

Reply:
I'll do some refactoring later and move `JsUtils::JsToJson` to common and then convert this over to use that.